empress/_plot.py

```diff
@@ -55,7 +56,8 @@ def plot(output_dir: str, tree: NewickFormat, feature_table: pd.DataFrame,
                   sample_metadata=sample_metadata,
                   feature_metadata=feature_metadata, ordination=pcoa,
                   ignore_missing_samples=ignore_missing_samples,
-                  filter_missing_features=filter_missing_features)
+                  filter_missing_features=filter_missing_features,
+                  filter_unobserved_features_from_phylogeny=filter_unobserved_features_from_phylogeny)
```

Comment:
```suggestion
                  filter_unobserved_features_from_phylogeny=(
                      filter_unobserved_features_from_phylogeny
                  )
```

(I think this should fix flake8?)

Reply:
fixed